api/routes/userRouter.js

```diff
@@ -4,22 +4,27 @@ const userModel = require('../models/userModel');
 
 
 router.get('/:id', (req, res, next) => {
-
+    
 });
 
 router.post('/', async (req, res, next) => {
     try {
-        const {data} = req.body;
-
-        const requiredParams = ['username', 'password', 'first_name', 'last_name', 'phone'];
-        const missingRequiredParams = requiredParams.filter(param => !(param in req.body));
-
-        if (missingRequiredParams.length > 0) {
-            throw new Error("Missing required user parameters: " + missingRequiredParams.join(', '));
+        const createParams = ['username', 'password', 'first_name', 'last_name', 'phone'];
+        const signInParams = ['username', 'password'];
+
+        const missingCreateParams = createParams.filter(param => !(param in req.body));
+        const missingSignInParams = signInParams.filter(param => !(param in req.body));
+
+        if(missingCreateParams.length == 0){
+            user = await userModel.createUser(req.body);
+            res.status(200).json({ user_id: user.id, message: "User created successfully with id " + user.id + "." });
```

Comment:
What does `.status(200)` do? Does that force the result to have a status of 200?

Reply:
It just sets the status code of the response to 200